src/IssueViz.Security/OpenInIDE/Http/HttpListenerFactory.cs

```diff
@@ -74,19 +89,21 @@ IDisposable IListenerFactory.Create(int startPort, int endPort)
             return listener;
         }
 
-        private static IDisposable CreateListener(int port)
+        private IDisposable CreateListener(int port)
         {
             var appProperties = new AppProperties(new Dictionary<string, object>())
             {
                 Addresses = AddressCollection.Create()
             };
 
+            // We're not specifying a URL so 
```

Comment:
Is this comment complete? It seems cut-off

Reply:
Have a look at the updated version; I think all of your comments have been addressed.